airflow/models/taskinstance.py

```diff
@@ -2128,6 +2138,14 @@ def set_duration(self) -> None:
             self.duration = None
         self.log.debug("Task Duration set to %s", self.duration)
 
+    def _record_task_map_for_downstreams(self, value: Any, *, session: Session) -> None:
+        if not self.task.has_mapped_dependants():
+            return
+        if not isinstance(value, collections.abc.Collection) or isinstance(value, (bytes, str)):
+            self.log.info("Failing %s for unmappable XCom push %r", self.key, value)
+            raise UnmappableXComPushed(value)
```

Comment:
I don't think we should include the _whole_ value here -- it could potentially be a huge object/structure/etc. (Either here, or in the exception itself.

Reply:
Changed both logging and exception to only include the variable type instead.